modules/registry-operator-configuration-resource-overview-aws-s3.adoc

```diff
@@ -1,12 +1,14 @@
 // Module included in the following assemblies:
 //
-// * openshift_images/configuring-registry-operator.adoc
+// * registry/configuring-registry-storage-aws-user-infrastructure.adoc
 
 [id="registry-operator-configuration-resource-overview-aws-s3_{context}"]
 = Image Registry Operator configuration parameters for AWS S3
 
 The following configuration parameters are available for AWS S3 registry storage.
 
+`ImageRegistryConfigStorageS3` holds the information to configure the registry to use the AWS S3 service for backend storage. See the link:https://docs.docker.com/registry/storage-drivers/s3/[S3 storage driver documentation] for more information.
```

Comment:
ISG wants "backend" to be two words, either "back-end storage" or "back end storage" should work. Or you could just write, "for server storage."